script/c511000685.lua

```diff
@@ -1,71 +1,71 @@
---ＷＲＵＭ－ホープ・フォース
```

Comment:
Did you change the line endings in this file?

Reply:
Nope, i didn't. I do not know why that happened